qb/select.go

```diff
@@ -163,3 +163,44 @@ func (b *SelectBuilder) AllowFiltering() *SelectBuilder {
 	b.allowFiltering = true
 	return b
 }
+
+// Func sets aggregation function for select clause on the query.
+func (b *SelectBuilder) Func(name, column string) *SelectBuilder {
+	b.Columns(name + "(" + column + ")")
```

Comment:
How about invoking append directly `b.columns = append(b.columns, name + "(" + column + ")")`, also `fmt.Sprint` would be better here.

Reply:
This will break encapsulation principle, cause `b.columns = append(b.columns, name + "(" + column + ")")` - is internal realization and `b.Columns()` - is external API that should be stable or bacward compatible...

Reply:
And I think, that with `b.Columns()` this will be more supportable.
But if you insist, I will change it to `append`.